roles/terrible/tasks/assert_vars.yml

```diff
@@ -294,7 +294,7 @@
   assert:
     quiet: yes
     that:
-      - "{{ item }}.encryption == True or {{ item }}.encryption == False"
+      - "{{ item }}.encryption or not {{ item }}.encryption"
```

Comment:
Did you test this statement? The reason I used the previous notation was that using your, I was not able to trigger the assert.

Reply:
Now it's fixed without doing literals